wizeline/fox-poc-backend
# Feat/add swagger doc

### What does this PR do?
Updates the `yaml` file for swagger documentation and adds the validation for `put` operations

### Where should the reviewer start?
At `docs/swagger.yaml` and `src/services/posts/posts.hooks.js`

### How can this be manually tested?
go to path `/docs` and review the documentation. Also, send a`put` operation without all the required fields and check that the response is an error
